js/src/components/LessonGrid.js

```diff
@@ -19,15 +19,26 @@ const useStyles = makeStyles((theme) => ({
       marginRight: 'auto',
     },
   },
+  innerIgnorePaddings: {
+    marginRight: theme.spacing(3.75),
+    [theme.breakpoints.down('md')]: {
+      marginRight: theme.spacing(2),
+    },
+  },
 }));
 
-const LessonGrid = ({ children }) => {
+const LessonGrid = ({ children, ignorePaddings = false }) => {
```

Comment:
This is not how you pass default properties. You need to refine them in LessonGrid.defaultProps = { };